openclaw_hooks/nima-recall-live/ladybug_recall.py

```
@@ -0,0 +1,364 @@
+        return memories, elapsed
+    except Exception as e:
+        print(f"[ladybug_recall] Get related error: {e}", file=sys.stderr)
+        return [], 0
+
+# =============================================================================
+# COMPATIBILITY LAYER (drop-in replacement for lazy_recall_v3)
+# =============================================================================
+
+def lazy_recall(query: str, 
+                 query_embedding: Optional[List[float]] = None,
+                 who: Optional[str] = None,
```

Ruff (0.15.0)

[warning] Unused function argument: `who`

(ARG001)